jscomps/PromptService.js

```diff
@@ -723,7 +772,20 @@ var PromptUtils = {
       realm += this.ellipsis;
     }
 
-    return realm;
+    let text;
```

Comment:
nitpick: I think I would prefer "textKey" as the variable name

Reply:
Yeah, agreed, it's better to make it clear.